store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedAdminIT.java

```diff
@@ -55,14 +54,14 @@
 
     @Override
     protected Schema createSchema() {
-        final Schema.Builder schemaBuilder = new Schema.Builder(createDefaultSchema());
+        final Schema.Builder schemaBuilder = new Schema.Builder(AbstractStoreIT.createDefaultSchema());
         schemaBuilder.edges(Collections.EMPTY_MAP);
         schemaBuilder.entities(Collections.EMPTY_MAP);
         return schemaBuilder.build();
     }
 
-    @Before
-    public void setUp() throws Exception {
+    @Override
```

Comment:
Does this method ALSO need @Before like the issue we have all be having with @Test.